city_scrapers/spiders/friant_water_authority.py

```diff
@@ -0,0 +1,124 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, COMMITTEE, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class FriantWaterAuthoritySpider(CityScrapersSpider):
+    name = "friant_water_authority"
```

Comment:
could you add fre_ as a prefix here, there are some upstream issues that make that required for now

Reply:
Gotcha. I've added the prefix in the latest commit of this PR. Please let me know if I missed anything else!